portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +23,11 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  private String name;
```

Comment:
If this should be a hardcoded string, change to private static final string and assign the value here (instead of in #doGet) so that it cannot be set again. Also, the convention is to have private static final variables have all caps with underscores (ALL_CAPS), so please change to NAME.